### ISSUE TITLE
Using JFoenix

### ISSUE THREAD

**Issue by durini309** [jfoenixadmin/JFoenix#85]
Hi! I'm new using java (with netbeans) and I've done everything this tutorial says (except for the gradle thing). I've include jfoenix.jar to my JavaFX Application, but now I don't know how to start using all this beautiful components. Can someone help me please? thank you!

**Comment by AshrafKACIMI** [comment]
I recommand that you learn how to make JavaFX applications, everything will be clear then.
There are basically two main way to make GUI. Either you use pure java code for both your component definition and your applicaiton logic, or you can use fxml for the first and java for the second.

I recommand the second way with the WYSIWYG tool "Scene Builder". You should then download Scene Builder and add JFoenix to it like the following picture:

![scenebuilder import](https://cloud.githubusercontent.com/assets/7318839/14581481/6580b51a-03e6-11e6-85bf-d7e7e0b66601.png)

You can then add JFoenix components from Scene Builder.

So basically I recommand that you learn how to build JavaFX applications and then it will be very easy to understand how to use JFoenix.

**Comment by durini309** [comment]
Hi! Thanks for yout quick answer! I figure out (with all demos included) how to add an specific component to my UI by code. Now, I'll try to lear more of javaFX. Thanks for your advice!

**Comment by Samehadel** [comment]
Please i have imported the jar file and i uses these beautiful components but i have a problem the JFXDatePicker can't be draged with me , anyone can help me ? 
Thank .
